test/unit/alphabet/nucleotide/rna15_test.cpp

```diff
@@ -11,79 +11,87 @@
+    EXPECT_EQ(seqan3::to_char(seqan3::rna15{}.assign_char('R')), 'R');
+    EXPECT_EQ(seqan3::to_char(seqan3::rna15{}.assign_char('Y')), 'Y');
+    EXPECT_EQ(seqan3::to_char(seqan3::rna15{}.assign_char('S')), 'S');
+    EXPECT_EQ(seqan3::to_char(seqan3::rna15{}.assign_char('W')), 'W');
+    EXPECT_EQ(seqan3::to_char(seqan3::rna15{}.assign_char('K')), 'K');
+    EXPECT_EQ(seqan3::to_char(seqan3::rna15{}.assign_char('M')), 'M');
+    EXPECT_EQ(seqan3::to_char(seqan3::rna15{}.assign_char('B')), 'B');
+    EXPECT_EQ(seqan3::to_char(seqan3::rna15{}.assign_char('D')), 'D');
+    EXPECT_EQ(seqan3::to_char(seqan3::rna15{}.assign_char('H')), 'H');
+    EXPECT_EQ(seqan3::to_char(seqan3::rna15{}.assign_char('V')), 'V');
 
-    EXPECT_EQ(to_char(rna15{}.assign_char('N')), 'N');
-    EXPECT_EQ(to_char(rna15{}.assign_char('!')), 'N');
+    EXPECT_EQ(seqan3::to_char(seqan3::rna15{}.assign_char('N')), 'N');
+    EXPECT_EQ(seqan3::to_char(seqan3::rna15{}.assign_char('!')), 'N');
 }
 
 TEST(rna15, char_literal)
 {
-    EXPECT_EQ(to_char('A'_rna15), 'A');
-    EXPECT_EQ(to_char('C'_rna15), 'C');
-    EXPECT_EQ(to_char('G'_rna15), 'G');
+    EXPECT_EQ(seqan3::to_char('A'_rna15), 'A');
+    EXPECT_EQ(seqan3::to_char('C'_rna15), 'C');
+    EXPECT_EQ(seqan3::to_char('G'_rna15), 'G');
 
-    EXPECT_EQ(to_char('U'_rna15), 'U');
-    EXPECT_EQ(to_char('T'_rna15), 'U');
+    EXPECT_EQ(seqan3::to_char('U'_rna15), 'U');
+    EXPECT_EQ(seqan3::to_char('T'_rna15), 'U');
 
-    EXPECT_EQ(to_char('R'_rna15), 'R');
-    EXPECT_EQ(to_char('Y'_rna15), 'Y');
-    EXPECT_EQ(to_char('S'_rna15), 'S');
-    EXPECT_EQ(to_char('W'_rna15), 'W');
-    EXPECT_EQ(to_char('K'_rna15), 'K');
-    EXPECT_EQ(to_char('M'_rna15), 'M');
-    EXPECT_EQ(to_char('B'_rna15), 'B');
-    EXPECT_EQ(to_char('D'_rna15), 'D');
-    EXPECT_EQ(to_char('H'_rna15), 'H');
-    EXPECT_EQ(to_char('V'_rna15), 'V');
+    EXPECT_EQ(seqan3::to_char('R'_rna15), 'R');
+    EXPECT_EQ(seqan3::to_char('Y'_rna15), 'Y');
+    EXPECT_EQ(seqan3::to_char('S'_rna15), 'S');
+    EXPECT_EQ(seqan3::to_char('W'_rna15), 'W');
+    EXPECT_EQ(seqan3::to_char('K'_rna15), 'K');
+    EXPECT_EQ(seqan3::to_char('M'_rna15), 'M');
+    EXPECT_EQ(seqan3::to_char('B'_rna15), 'B');
+    EXPECT_EQ(seqan3::to_char('D'_rna15), 'D');
+    EXPECT_EQ(seqan3::to_char('H'_rna15), 'H');
+    EXPECT_EQ(seqan3::to_char('V'_rna15), 'V');
 
-    EXPECT_EQ(to_char('N'_rna15), 'N');
-    EXPECT_EQ(to_char('!'_rna15), 'N');
+    EXPECT_EQ(seqan3::to_char('N'_rna15), 'N');
+    EXPECT_EQ(seqan3::to_char('!'_rna15), 'N');
 }
 
 TEST(rna15, string_literal)
 {
-    rna15_vector v;
+    seqan3::rna15_vector v;
     v.resize(5, 'A'_rna15);
     EXPECT_EQ(v, "AAAAA"_rna15);
 
-    std::vector<rna15> w{'A'_rna15, 'C'_rna15, 'G'_rna15, 'T'_rna15, 'U'_rna15, 'N'_rna15};
+    std::vector<seqan3::rna15> w{'A'_rna15, 'C'_rna15, 'G'_rna15, 'T'_rna15, 'U'_rna15, 'N'_rna15};
     EXPECT_EQ(w, "ACGTTN"_rna15);
 }
 
 TEST(rna15, char_is_valid)
 {
```

Comment:
I see why you decide to add some additional new lines, but I would still prefer 3 characters for each line. 
Otherwise I agree with everything. Approved.